package.json

```diff
@@ -29,8 +29,12 @@
     "install": "node-pre-gyp install --fallback-to-build"
   },
   "dependencies": {
-    "nan": "2.8.0",
+    "nan": "2.9.2",
+<<<<<<< HEAD
     "node-pre-gyp": "0.7.0"
+=======
+    "node-pre-gyp": "0.6.39"
+>>>>>>> pin version of nan
```

Comment:
Remnants from merge conflicts?

Reply:
D'oh. Sorry about that. Not sure how that happened. All fixed now.